src/Symfony/Component/Process/InputStream.php

```diff
@@ -20,12 +20,15 @@
  */
 class InputStream implements \IteratorAggregate
 {
+    /** @var null|callable */
```

Comment:
Was added to avoid notice `callable expected currently null`

Even with the check on `null !== $onEmpty` this message is still popping up.
This is becuase the IDE does not recognize the type of `$this->onEmpty` given from the constructor.

`$this->write($onEmpty(...))`